chapters/ch17.md

```diff
@@ -1,32 +1,33 @@
 ->
-
+Hãy cùng tiếp tục với ví dụ ở trên: ví dụ thuật toán đang phân loại nhầm 1000 mẫu trên tổng số 5000 mẫu trong tập phát triển. Giả sử chúng ta muốn sử dụng 100 ví dụ bị phân lọại nhầm để phân tích lỗi (10% tổng số lỗi). Bạn nên chọn ra 10% mẫu trong tập phát triển một cách ngẫu nhiên và đặt nó vào trong một tập mà chúng ta sẽ gọi là **tập phát triển nhãn cầu** để nhắc chúng ta rằng ta sẽ trực tiếp nhìn vào bằng mắt. (Đối với những dự án nhận diện giọng nói mà bạn phải nghe audio, có lẽ bạn sẽ gọi tập này là tập phát triển tai). Tập phát triển nhãn cầu chứa 500 ví dụ, trong đó chúng ta sẽ kỳ vọng thuật toán sẽ phân loại nhầm khoảng 100 ví dụ.
 
 The second subset of the dev set, called the **​Blackbox dev set​**, will have the remaining 4500 examples. You can use the Blackbox dev set to evaluate classifiers automatically by measuring their error rates. You can also use it to select among algorithms or tune hyperparameters. However, you should avoid looking at it with your eyes. We use the term “Blackbox” because we will only use this subset of the data to obtain “Blackbox” evaluations of classifiers.
 ->
-
+Tập thứ hai của tập phát triển, được gọi là **tập phát triển hộp đen**, sẽ chứa 4500 mẫu còn lại. Bạn có thể sử dụng tập phát triển hộp đen để đánh giá các bộ phân loại một cách tự động bằng cách đo tỉ lệ lỗi chúng. Bạn cũng có thể sử dụng tập này để lựa chọn thuật toán hoặc điều chỉnh tham số. Tuy nhiên, bạn nên tránh trực tiếp phân tích thủ công tập này. Chúng ta sử dụng thuật ngữ "hộp đen" bời vì chúng ta chỉ sử dụng tập này để thu về những đánh giá "hộp đen" của bộ phân loại. 
 
 ![img](../imgs/C17_02.png)
 
 Why do we explicitly separate the dev set into Eyeball and Blackbox dev sets? Since you will gain intuition about the examples in the Eyeball dev set, you will start to overfit the Eyeball dev set faster. If you see the performance on the Eyeball dev set improving much more rapidly than the performance on the Blackbox dev set, you have overfit the Eyeball dev set. In this case, you might need to discard it and find a new Eyeball dev set by moving more examples from the Blackbox dev set into the Eyeball dev set or by acquiring new labeled data.
 ->
-
+Tại sao chúng ta lại chia tập phát triển thành tập phát triển nhãn cầu và tập phát triển hộp đen? Khi bạn nhận thêm trực giác về những mẫu trong tập phát triển nhãn cầu, bạn sẽ overfit tập phát triển đó nhanh hơn. Khi bạn thấy chất lượng của tập phát triển nhãn cầu đang tăng nhanh hơn nhiều so với tập phát triển hộp đen, bạn đã overfit tập phát triển nhãn cầu. Trong trường hợp này, bạn thể phải loại bỏ tập phát triển này đi, và tìm một tập khác thay thế bằng cách chuyển các mẫu từ tập phát triển hộp đen sang tập phát triển nhãn cầu, hoặc thu về những mẫu được gán nhãn khác. 
```

Comment:
à theo mình hiểu thì không hẳn là loại bỏ hẳn đi mà dùng tập black box sample ra tập Eyeball mới, còn tập cũ thì có thể cho lại tập black box, vì nếu bỏ đi hẳn 1 phần training example có vẻ hơi vô lý. Đoạn này đọc lại đúng là mình dịch không rõ thật, có lẽ đợi xem mọi người cho ý kiến về ý của tác giả rồi dịch lại 1 thể.
Cái phần acquire new labeled data mình cũng không rõ lắm, theo mình thì là lấy data ngoài tập dev sets.

Reply:
@khoapip : Eyeball dev set cũ cũng chỉ là dev set thôi bạn; khi sử dụng nó cho error analysis + thay đổi parameters của model & model đã overfit Eyeball dev set này rồi thì giữ nó hay bỏ nó đi cũng giống nhau.

Reply:
Theo e thấy, ở đây, bác tác giả có nói là chia ra thành Eyeball và Blackbox. Và chỉ sử dụng Eyeball để cho error analysis chứ không dùng Blackbox như tiêu đề chương, vì Eyeball phải làm sao đó vừa đủ cho có thể phân tích thủ công và đủ để đại diện cho tập dev nên mới xảy ra vấn đề phải chọn Eyeball sao cho không overfit. Còn đoạn này "discard it" e nghĩ "loại bỏ tập Eyeball", thay vào một tập Eyeball mới lấy từ Blackbox hoặc từ dữ liệu được gán nhãn mới.

Reply:
Mình thấy thiếu từ "explicit" ám chỉ hai tập này không có overlap 

Câu "since you will gain intuition ..." hiểu theo nghĩa "khi bạn hiểu rõ hơn về .." thì hay hơn từ "trực quan" . 

Câu cuối ám chỉ lấy thêm dữ liệu có nhãn chứ ko phải lấy từ tập khác.

Reply:
@ngcthuong : đồng ý là lấy thêm dữ liệu có nhãn, nhưng dữ liệu có nhãn này lấy từ đâu cậu (assume là cùng distribution với dev set)?

Reply:
Oh, PR này chú ý cách dịch Eyeball và Blackbox nhé, xem thêm tại #194.

Reply:
@william-vu có lẽ sample từ training set ra thôi, đoạn này mình thấy phần source cũng khá mập mờ. 
@tiepvupsu ok a, lúc đầu em dịch vẫn chưa thống nhất được 2 từ này.